### ISSUE TITLE
Question: No external access / local VPN setup

### ISSUE THREAD

**Issue by tobyweston** [pivpn/pivpn#245]
Hi, 

I've setup pivpn and am trying to connect my machine via tunnelblink but 

My client config shows the following but I haven't setup any port forwarding from `71.42.110.101` so I can't get tunnelblink to connect.

    remote 71.42.110.101 1194

I understand the above IP to be my externally facing IP address. However, I'm not really interested in opening a port to the outside world, I'm mostly interested in routing traffic from machines on my local network through openVPN to get encrypted/anonymous traffic once out on the web.

Is this a common use case or am I swimming against the tide here?

In summary, should I be using pivpn/openVPN if I don't need to connect to my home network from the internet but instead just want anonymous surfing from within? Can I set things up without port forwarding and just route locally?

Thanks in advance

**Comment by cfcolaco** [comment]
hello, @tobyweston there is no point in using a VPN withing your home network. 

The purpose of a VPN is to safely connect to any given network from an unsafe one. 

For example, you are at a hotel, and you want to connect to your home, that way, you hide your traffic from prying eyes, you can access to devices on your home network, and your externally facing IP address changes, instead of having the hotel IP address you start having your Home IP address, which allows you to bypass country restrictions if, for example, you are in a country with heavy internet restrictions. 

it is pointless to route all your home network through a home VPN.

if you want some privacy **from** your home through the internet, you should have a VPN somewhere outside your home, there are some free VPNs (most of them are not reliable), some that are paid and have a great reputation, or you can just buy a VPS and setup OpenVPN on that VPS ( which require a bit more of knowledge but is not that hard). 

Hope I've been clear to you, however, and since this is just a superficial explanation, I still recommend you to research a little more about VPN and HTTPS, so you can have a better idea of how they work.

**Comment by tobyweston** [comment]
Thanks for the reply. That's cleared it up for me a little. I was thinking I could use the VPN as a gateway to the internet like an anonymizing proxy I guess.
